todolist.html

```diff
@@ -32,5 +32,24 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+
+        <div>
+            <div>
+                <header>
+                    Delete Task?
+                </header>
+                <div>
+                    <p>Are you sure you want to delete this task?</p>
+                </div>
+                <footer>
+                    <button type='button'>
+                        Delete
+                    </button>
+                    <button type='button'>
+                        Cancel
+                    </button>
+                </footer>
+            </div>
```

Comment:
Maybe remove one of these outer divs. I know we will have to add it back when we incorporate styles, but for right now it looks pointless.